### PR TITLE
Replaced Promise.each() with .all() in models/base/bookshelf

### PR DESCRIPTION
**Pull request by CarolBarno**
Got some code for us? Awesome 🎊!

Please include a description of your change & check your PR against this list, thanks!

- [ ] There's a clear use-case for this code change, explained below
- [ ] Commit message has a short title & references relevant issues
- [ ] The build will pass (run `yarn test:all` and `yarn lint`)

We appreciate your contribution! 

Also, if you'd be interested in writing code like this for us more regularly, we're hiring:
https://careers.ghost.org/product-engineer-node-js

### CODE REVIEW

File: ghost/core/core/server/models/base/bookshelf.js [hunk 1]
```diff
@@ -84,13 +83,13 @@ ghostBookshelf.plugin('bookshelf-relations', {
                     return Promise.resolve();
                 }
 
-                return Promise.each(targets.models, function (target, index) {
```
Comment (ErisDS) [inline ghost/core/core/server/models/base/bookshelf.js:87]: This change doesn't look right to me. Promise.all takes an array of iterables

Comment (ErisDS) [inline ghost/core/core/server/models/base/bookshelf.js:87]: This change doesn't look right to me. Promise.all takes an array of iterables

**Review by ErisDS — CHANGES_REQUESTED** [review]

**Review by ErisDS — APPROVED** [review]

**Comment by ErisDS** [comment]
Hi @CarolBarno thanks again for this PR and for taking the time to update it. Sorry for the delay (I got a little swamped with PRs) but it has now been merged 🎉 and will appear in the next release of Ghost - usually Fridays.

I'm not sure if you found this through hacktoberfest, but I've added the accepted label to this PR to make sure it counts. 

There are many more places where Promise.map is used, would be very grateful for more PRs like this (and faster at merging I _promise_!) ❤️

**Comment by CarolBarno** [comment]
Hi @ErisDS , thanks for your response and let me look at similar places that require replacing.

**Pull request merged**